src/analysis/object.rs

```diff
@@ -97,6 +98,9 @@ pub fn class(env: &Env, obj: &GObject, deps: &[library::TypeId]) -> Option<Info>
     if obj.generate_display_trait {
         imports.add("std::fmt", None);
     }
+    if obj.subclassing {
+        imports.add(&format!("{}Class", name), None);
+    }
```

Comment:
Maybe move down to `rust_class_type` and combine?

Reply:
Thanks, done!